src/Symfony/Component/Serializer/Encoder/CsvEncoder.php

```diff
@@ -281,4 +294,18 @@ private function extractHeaders(iterable $data): array
 
         return $headers;
     }
+
+    private function determineBom(string $text): string
```

Comment:
This is copied from https://github.com/thephpleague/csv/blob/0b843fb2578371259e12722c6c0edf28625faa43/src/functions.php#L33 - shall I attribute this somehow?